frontend/src/components/StoreList/StoreList.js

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
-
+import React, { useState, useEffect } from 'react';
 import { businessImages } from '../../shared/businessImages'
 import './StoreList.scss';
 import { Link } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
+import API from '../../shared/api';
 
 
 const StoreList = () => {
   const [t] = useTranslation('store-list');
-  const stores = [
-    { name: 'Manfreds Bakery', desc: 'A mix of some flourish stuff and water, plus some salty crystals looking like salt...' },
-    { name: 'BlueMen', desc: 'A mix of some flourish stuff and water, plus some salty crystals looking like salt...' },
-    { name: 'DanyOils', desc: 'A mix of some flourish stuff and water, plus some salty crystals looking like salt...' },
-  ];
+  const [stores, setStores] = useState([])
+
+  useEffect(() => {
+    const getStores = async () => {
+      let partners = []
+      const response = await API.partners.get()
+      response.data.map(store => {
+        return partners.push({
+          id: store.id,
+          name: store.name,
+          desc: store.description
+        })
+      })
+      setStores(partners);
+    }
+    getStores();
+
+  })
```

Comment:
Please check this does not end up in a loop, because that code will be running every render